File: area/area.cpp
#include <iostream>
using namespace std;

int main(){
    short unsigned int lato, figura;
    float risultato;
    cout << "Scegli di che figura vuoi sapere l'area\n1 Quadrato\n2 Cerchio\n3 Triangolo equilatero\n";
    cin >> figura;
    cout << "Inserisci il lato o raggio\n";
    cin >> lato;
    if(figura == 1){
        risultato = lato * lato;
    } else {
        if(figura == 2){
            risultato = (float)((lato * lato) * 3,14);
        } else {
            risultato = (float)((lato * lato) / 2);
        }
    }
    cout << "Il risultato e ";
    cout << risultato;
    cout << "\n"; 
}

File: Java/Area/area.java
import java.util.Scanner;

public class area{
    public static void main(String[] args){
        int lato, figura;
        float risultato;
        System.out.println("Scegli di che figura vuoi sapere l'area\n1 Quadrato\n2 Cerchio\n3 Triangolo equilatero\n");  
        Scanner scanIn =new Scanner(System.in); //inzializza lo scanIn, da chiudere quando finisce il programma
        figura = scanIn.nextInt(); 
        System.out.println("Inserisci il lato o il raggio");
        lato = scanIn.nextInt();
        if(figura == 1){
            risultato = (float)(lato * lato);
        } else{
            if(figura == 2){
                risultato = (float)((lato * lato) * 3.14);
            } else {
                risultato = (float)((lato * lato) / 2);
            }
        }
        System.out.println("il risultato è " + risultato);
        //in.close(); //chiude lo scanIn 
    }

}